src/main/java/com/example/sortener/controller/ShortenerController.java

```diff
@@ -0,0 +1,48 @@
+package com.example.sortener.controller;
+
+import com.example.sortener.dto.shortener.RequestUrlDto;
+import com.example.sortener.dto.shortener.ResponseUrlDto;
+import com.example.sortener.service.ShortenerService;
+import com.example.sortener.validator.ApplicationValidator;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.slf4j.Logger;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import static java.lang.System.currentTimeMillis;
+import static org.slf4j.LoggerFactory.getLogger;
+import static org.springframework.http.ResponseEntity.ok;
+
+@RestController
+@RequestMapping
+@EnableWebMvc
+@RequiredArgsConstructor
+public class ShortenerController {
+
+    @NonNull
```

Comment:
It's injected via dependency injection, which wont allow app to start if it cant find bean, which means it wont start if bean is null, so do you need check for non-null here?